lib/Runtime/Base/FunctionInfo.h

```diff
@@ -44,6 +44,7 @@ namespace Js
         };
         FunctionInfo(JavascriptMethod entryPoint, Attributes attributes = None, LocalFunctionId functionId = Js::Constants::NoFunctionId, FunctionProxy* functionBodyImpl = nullptr);
         FunctionInfo(JavascriptMethod entryPoint, _no_write_barrier_tag, Attributes attributes = None, LocalFunctionId functionId = Js::Constants::NoFunctionId, FunctionProxy* functionBodyImpl = nullptr);
+        FunctionInfo(FunctionInfo& that); // Todo: (leish)(swb) find a way to prevent non-static initializer calling this ctor
```

Comment:
How is this constructor used? Does it help to add a dummy ", _no_write_barrier_tag" parameter so it must be called explicitly as "FORCE_NO_WRITE_BARRIER_TAG(that)"?